packages/dds/merge-tree/src/attributionPolicy.ts

```diff
@@ -209,6 +214,7 @@ export function createInsertOnlyAttributionPolicy(): AttributionPolicy {
 }
 
 /**
+ * Creates an AttributionPolicy for tracking annotation of specific properties.
```

Comment:
Nit
```suggestion
 * Creates an {@link AttributionPolicy} for tracking annotation of specific properties.
```

(Should get auto-updated if `AttributionPolicy` gets renamed with refactoring tools.